src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunner.cs

```diff
@@ -508,7 +498,7 @@ private string GenerateRunSettings(int? timeout, bool forMutantTesting, bool for
 
             if (_testFramework.HasFlag(TestFramework.xUnit))
             {
-                settingsForCoverage += "<DisableParallelization>true</DisableParallelization>";
+                settingsForCoverage += "<Disable>true</DisableParallelization>";
```

Comment:
Here is the bug, the xml tags are inconsistently replaced

Reply:
Looks more promissing:

> [17:19:43 INF] 242   mutants got status CompileError. Reason: Mutant caused compile errors
[17:19:43 INF] **409**   mutants got status NoCoverage.   Reason: Mutant has no test coverage
[17:19:43 INF] 1     mutants got status Ignored.      Reason: Unable to inject back mutations in source code.
[17:19:43 INF] 652   total mutants are skipped for the above mentioned reasons
[17:19:43 INF] **9400**  total mutants will be tested

Reply:
> 100,00% │ Testing mutant 9400 / 9400 │ K 8295 │ S 1020 │ T 85 │ ~0m 00s │
> 
> Killed:   8295
Survived: 1020
Timeout:    85
> 
> [18:38:30 INF] Time Elapsed 01:20:44.0972875
[18:38:30 INF] The final mutation score is 85.43 %

Reply:
Thanks for spotting this. This is indeed some mishap from my part. This is a xUnit specific settings, and NFluent uses NUnit. I understand why I was not able to reproduce it.

Reply:
as per the other conversation, this problem should have been detected by integration tests. It turns out they have been disabled for several months 😄. Thanks for your contribution that helped us fix two significant issues.